### PR TITLE
Weapon Dealer remake

### PR DESCRIPTION
**Pull request by CnRXoMoX**

### CODE REVIEW

File: gamemodes/core/admin/admin_account.inc [hunk 1]
```diff
@@ -159,4 +159,20 @@ stock Admin_IsOnDuty(playerid)
 timer KickTimer[500](playerid)
 {
     Kick(playerid);
+}
+
+stock SendAdminCmdUsage(playerid, const cmd[], const cmdparam[])
```
Comment (PatrickGTR) [inline gamemodes/core/admin/admin_account.inc:164]: can you remove this?

File: gamemodes/core/player/job/job_weapon-dealer-table.inc [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#include <YSI_Coding\y_hooks>
+
+hook OnMySQLConnected()
+{
+    static const query[] = "\
+    CREATE TABLE IF NOT EXISTS player_weaponInv \
```
Comment (PatrickGTR) [inline gamemodes/core/player/job/job_weapon-dealer-table.inc:6]: can we stick small letters, snake_case naming convention.

e.g

weapon_Inventory_player

**Review by PatrickGTR — CHANGES_REQUESTED** [review]

File: gamemodes/core/admin/admin_account.inc (hunk 1 above)
Comment (CnRXoMoX) [reply gamemodes/core/admin/admin_account.inc:164]: Sure

**Comment by CnRXoMoX** [comment]
All done.

**Pull request merged**